app/scripts/views/mixins/migration-mixin.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Helper functions for views with migration-specific behaviour.
+// Meant to be mixed into views.
+
+define([
+  'lib/url',
+], function (Url) {
+  'use strict';
+
+  return {
+    isMigration: function () {
+      return !! Url.searchParam('migration', this.window.location.search);
```

Comment:
We have slowly been moving away from the direct use of search parameters in the views because we found that importing data from the search params was inflexible when we wanted to use alternative data sources like the [ResumeToken](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/resume-token.js). Instead we have been using the AuthBroker and Relier models. All views have a reference to a `relier` model. We import `migration` into the [FxDesktopRelier](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/reliers/fx-desktop.js#L39). Similar to how we have [isFxDesktop](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/reliers/fx-desktop.js#L56), perhaps a new function `isMigration` could be used, or maybe line 15 could just read:

```js
   return this.relier.has('migration');
```